shared/common-adapters/usernames.js

```diff
@@ -22,6 +22,8 @@ function usernameText({
   colorBroken = true,
   onUsernameClicked,
   underline = false,
+  inlineGrammar = false,
```

Comment:
figured I would just add these here instead of making a one-off for inline lists of usernames